### DISCUSSION TITLE
OOP-IEC61131-3--Curso-Youtube/Clases%20y%20Objetos/clases%20y%20objetos/

### DISCUSSION THREAD

**Discussion by giscus** [runtimevic/OOP-IEC61131-3--Curso-Youtube#15] [General]
# OOP-IEC61131-3--Curso-Youtube/Clases%20y%20Objetos/clases%20y%20objetos/

IEC 61131-3 PLC OOP Course

https://runtimevic.github.io/OOP-IEC61131-3--Curso-Youtube/Clases%20y%20Objetos/clases%20y%20objetos/

**Comment by dadd86** [comment]
muchas gracias

**Reply by runtimevic** [reply]
Hola, gracias a ti, espero que te ayude con tu aprendizaje de OOP...
